go/enclave/db/storage.go

```diff
@@ -254,12 +257,20 @@ func (s *storageImpl) FetchLogs(hash common.L1RootHash) ([]*types.Log, error) {
 func (s *storageImpl) StoreNewHead(state *core.BlockState, rollup *core.Rollup, receipts []*types.Receipt, logs []*types.Log) error {
 	batch := s.db.NewBatch()
 
-	if state.FoundNewRollup {
-		obscurorawdb.WriteRollup(batch, rollup, s.logger)
-		obscurorawdb.WriteHeadHeaderHash(batch, rollup.Hash(), s.logger)
-		obscurorawdb.WriteCanonicalHash(batch, rollup.Hash(), rollup.NumberU64(), s.logger)
+	if state.FoundNewRollup { //nolint:nestif
+		if err := obscurorawdb.WriteRollup(batch, rollup); err != nil {
```

Comment:
maybe we could create an: `s.storeNewRollup()` to tidy these writes up under a readable label, and avoid the lint ignore.

Reply:
Done